lib/generators/solidus_stripe/install/install_generator.rb

```diff
@@ -78,6 +78,18 @@ def install_solidus_starter_frontend_support
               template engine.root.join(path), path
             end
           end
+
+          storefront_layout_path = Rails.root.join("app/views/layouts/storefront.html.erb")
+
+          if storefront_layout_path.exist?
+            insert_into_file(
+              storefront_layout_path.to_s,
+              "\n    <script async src='https://js.stripe.com/v3/'></script>",
```

Comment:
Why this approach?

Reply:
This comes straight outta `importmaps-rails` https://github.com/rails/importmap-rails/blob/main/lib/install/install.rb#L1-L9 and is in line with Stripe advising to load the JS on every page to better support fraud prevention. The stripe-js imported in .js files on the other hand is just a thin layer that will load this same file if it's not already available, so we can have the global script tag marked as `async` and at the same time use a comfy `import` in our JS files.